lib/internal/streams/destroy.js

```diff
@@ -339,6 +339,9 @@ function destroyer(stream, err) {
     stream.abort();
   } else if (isRequest(stream.req)) {
     stream.req.abort();
+  } else if (stream === process.stdout && typeof stream.end === 'function') {
```

Comment:
This is not a correct fix, ideally we should not check this here but rather fix whatever `process.stdout` is doing wrong.

Reply:
Also, isn't `stream.end` would make process.stdout unusable?

Reply:
Great, you know where the logic of process.stdout happens to fix them?

Reply:
> Also, isn't `stream.end` would make `process.stdout` unusable and maybe crash?

I need to investigate more

Reply:
I think I found the problem, see my latest comment

Reply:
Is this still needed?

Reply:
Yes to catch error on the callback